Documentation/data-collection.md

```diff
@@ -584,6 +584,14 @@ data:
     # monitoring routes.
     - '{__name__="monitoring:haproxy_server_http_responses_total:sum"}'
     #
+    # owners: (@openshift/openshift-team-monitoring)
+    #
+    # cluster_monitoring_operator_configured_collection_profile contains information about the last configured
+    # collection profile.
+    # Possible label values are:
+    #   collection_profile= full|minimal (refer: cluster-monitoring-operator/pkg/manifests#SupportedCollectionProfiles)
+    - '{__name__="cluster_monitoring_operator_configured_collection_profile"}'
```

Comment:
You want to aggregate and record the metric to remove variable/unnecessary labels such as pod, job, service and instance. E.g. `max by(profile) (cluster_monitoring_operator_collection_profile)`

https://rhobs-handbook.netlify.app/products/openshiftmonitoring/telemetry.md/#configure-recording-rules

Reply:
Ah! Apologies, I totally missed that. Addressed.